scully/routerPlugins/jsonRoutePlugin.ts

```diff
@@ -23,9 +23,9 @@ const jsonRoutePlugin = async (route: string, conf: RouteTypeJson): Promise<Hand
     const loadData = (param, context = {}): Promise<any[]> => {
       /** us es-template lie string to construct the url */
       const url = renderTemplate(conf[param.part].url, context);
-      return httpGetJson(url).then((rawData: any) =>
-        rawData.map(row => deepGet(conf[param.part].property, row))
-      );
+      return httpGetJson(url, {
+        headers: conf[param.part].headers,
```

Comment:
Does this error if there aren't any provided? Does it work without anything?

Reply:
hmm good question I currently don't know how to test it locally ^^ -- probably I need to add `|| {}` to it in case it's `null` or `undefined`. An empty object should be fine I think.